### PR TITLE
[MAINTENANCE] Edits (Formatting)

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md [hunk 1]
```diff
@@ -28,16 +28,16 @@ Steps
 
   ```yaml
```
Comment (cdkini) [inline docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Please delete this line :)

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md [hunk 2]
```diff
@@ -46,18 +46,18 @@ Steps
...
```
Comment (cdkini) [inline docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Please also delete this line :)

**Review by cdkini — CHANGES_REQUESTED** [review]
The code blocks look good but please fix the items that have not been resolved here: https://github.com/great-expectations/great_expectations/pull/3015

Let me know if you want to quickly pair if that would be easier.

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Done!

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md (hunk 2 above)
Comment (alexsherstinsky) [reply docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Done!

File: docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-terminal.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 -----
 
 1. Checkpoints can be run like applications from the command line by running:
 
 ```bash
-
-    great_expectations --v3-api checkpoint run my_checkpoint
-    Validation failed!
+  great_expectations --v3-api checkpoint run my_checkpoint
+  Validation failed!
```
Comment (cdkini) [inline docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-terminal.md]:
```suggestion
great_expectations --v3-api checkpoint run my_checkpoint
Validation failed!
```

Please remove whitespace before code block contents so they're all aligned.

File: docs/guides/validation/how-to-add-a-validation-operator.md [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
+- Created at least one [Checkpoint](./checkpoints/how-to-create-a-new-checkpoint). You will need it in order to test that your new Validation Operator is working.
 
-This how-to guide assumes you have already:
+</Prerequisites>
 
-  - Created at least one Expectation Suite.
-  - Created at least one [Checkpoint](./checkpoints/how-to-create-a-new-checkpoint). You will need it in order to test that your new Validation Operator is working.
+This guide will help you add a new instance of a [Validation Operator](../../reference/checkpoints-and-actions). Validation Operators give you the ability to encode business logic around validation, such as validating multiple batches of data together, differentiating between warnings and errors, and kicking off actions based on the results of validation.
+
+As a general rule, Validation Operators should be invoked from within [Checkpoints](../../reference/core-concepts#checkpoints). Separating out the configuration for Validation Operators and Checkpoints can help make Operator code reusable.
```
Comment (cdkini) [inline docs/guides/validation/how-to-add-a-validation-operator.md:22]: YAML block on L30 is not closing properly; please fix L51 to ensure code block closes. Currently, the rest of the document is treated as YAML due to this error.

File: docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md [hunk 5]
```diff
@@ -1,84 +1,81 @@
...
+- Created at least one [Checkpoint](./how-to-create-a-new-checkpoint)
 
-  - Created at least one [Checkpoint](./how-to-create-a-new-checkpoint).
+</Prerequisites>
 
 Steps
 -----
 
-1. First, generate the python with the command:
+1. First, generate the Python with the command:
```
Comment (cdkini) [inline docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md]: If we're capitalizing Python, can we ensure consistency within the entire document?

**Review by cdkini — CHANGES_REQUESTED** [review]
Just an issue or two to resolve and then we're good!

File: docs/guides/validation/how-to-add-a-validation-operator.md (hunk 4 above)
Comment (alexsherstinsky) [reply docs/guides/validation/how-to-add-a-validation-operator.md:22]: Done!

File: docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md (hunk 5 above)
Comment (alexsherstinsky) [reply docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md]: Done!

File: docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-terminal.md (hunk 3 above)
Comment (alexsherstinsky) [reply docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-terminal.md]: Done!

**Review by cdkini — APPROVED** [review]
LGTM! Thanks for all the updates 🎉

**Pull request merged**